cli/commands/cluster/member-remove.go

```diff
@@ -0,0 +1,36 @@
+package cluster
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/sensu/sensu-go/cli"
+	"github.com/spf13/cobra"
+)
+
+func MemberRemoveCommand(cli *cli.SensuCli) *cobra.Command {
+	return &cobra.Command{
+		Use:          "member-remove [ID]",
+		Short:        "remove cluster member by ID",
+		SilenceUsage: false,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) != 1 {
+				_ = cmd.Help()
+				return errors.New("invalid argument(s) received")
+			}
+
+			memberID := args[0]
+			id, err := strconv.ParseUint(memberID, 16, 64)
```

Comment:
AFAIK member IDs are not integers:

```
 $ ./bin/sensuctl cluster member-list
         ID            Name           Peer URLs              Client URLs
 ────────────────── ────────── ─────────────────────── ───────────────────────
  8927110dc66458af   backend1   http://127.0.0.1:2380   http://127.0.0.1:2379
  bb75bf8de77581cc   backend2   http://127.0.0.1:3380   http://127.0.0.1:3379
$ ./bin/sensuctl cluster member-remove bb75bf8de77581cc
Error: error removing cluster member: strconv.Atoi: parsing "bb75bf8de77581cc": invalid syntax
```

Reply:
Member IDs are hex encoded unsigned integers.

You're probably getting this error because I changed the encoding from decimal to hexadecimal in the backend, and you didn't rebuild. The reason I changed this was to bring us in line with what etcd does.

Reply:
I gotcha!